utxodump.go

```diff
@@ -4,9 +4,11 @@ package main
 import "github.com/in3rsha/bitcoin-utxo-dump/bitcoin/btcleveldb" // chainstate leveldb decoding functions
 import "github.com/in3rsha/bitcoin-utxo-dump/bitcoin/keys"   // bitcoin addresses
 import "github.com/in3rsha/bitcoin-utxo-dump/bitcoin/bech32" // segwit bitcoin addresses
+import "github.com/in3rsha/bitcoin-utxo-dump/writer" // segwit bitcoin addresses
```

Comment:
nit: the comment is misleading. it's importing the writer interface to dump the UTXOs in csv or parquet